fitters/src/DAF.cc

```diff
@@ -191,15 +211,16 @@ void DAF::addProbCut(const double prob_cut, const int measDim){
   chi2Cuts_[measDim] = ROOT::Math::chisquared_quantile_c( prob_cut, measDim);
 }
 
-void DAF::setAnnealingScheme(double bStart, double bFinal, unsigned int nSteps) {
+void DAF::setAnnealingScheme(double bStart, double bFinal, unsigned int nSteps) {}
```

Comment:
You should keep the old setAnnealingScheme, and add your new one. otherwise you force the genfit people to set minIter and maxIter, which are not present in the old constructor